frontend/javascripts/oxalis/model/sagas/save_saga.ts

```diff
@@ -988,33 +990,54 @@ export function performDiffTracing(
 
   return actions;
 }
+
+let isWkReady = false;
+
+function setWkReady() {
+  isWkReady = true;
+}
 export function* saveTracingAsync(): Saga<void> {
+  yield* takeEvery("WK_READY", setWkReady);
   yield* takeEvery("INITIALIZE_SKELETONTRACING", saveTracingTypeAsync);
   yield* takeEvery("INITIALIZE_VOLUMETRACING", saveTracingTypeAsync);
+  yield* takeEvery("INITIALIZE_EDITABLE_MAPPING", saveEditableMappingAsync);
+}
+
+export function* saveEditableMappingAsync(
```

Comment:
Not new due to this PR, but what do you think about renaming `saveEditableMappingAsync` and `saveTracingTypeAsync` to something like `setupSavingFor...`? I think, it would fit better to what is happening there. The current `save` sounds like it would happen once when called.